code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceCreate.java

```diff
@@ -92,33 +88,7 @@ private void createLabels(Instance instance) {
             String labelKey = labelEntry.getKey();
             String labelValue = labelEntry.getValue();
 
-            Label label = objectManager.findAny(Label.class,
-                    LABEL.KEY, labelKey,
-                    LABEL.VALUE, labelValue,
-                    LABEL.ACCOUNT_ID, instance.getAccountId(),
-                    LABEL.REMOVED, null);
-            if (label == null) {
-                Map<Object, Object> labelData = new HashMap<>();
-                labelData.put(LABEL.NAME, labelKey + "=" + labelValue);
-                labelData.put(LABEL.KEY, labelKey);
-                labelData.put(LABEL.VALUE, labelValue);
-                labelData.put(LABEL.ACCOUNT_ID, instance.getAccountId());
-                labelData.put(LABEL.TYPE, LabelConstants.HOST_TYPE);
-                label = resourceDao.createAndSchedule(Label.class, objectManager.convertToPropertiesFor(Label.class, labelData));
-            }
-
-            InstanceLabelMap mapping = objectManager.findAny(InstanceLabelMap.class,
-                    INSTANCE_LABEL_MAP.LABEL_ID, label.getId(),
-                    INSTANCE_LABEL_MAP.INSTANCE_ID, instance.getId(),
-                    INSTANCE_LABEL_MAP.ACCOUNT_ID, instance.getAccountId(),
-                    INSTANCE_LABEL_MAP.REMOVED, null);
-            if (mapping == null) {
-                Map<Object, Object> mappingData = new HashMap<>();
-                mappingData.put(INSTANCE_LABEL_MAP.LABEL_ID, label.getId());
-                mappingData.put(INSTANCE_LABEL_MAP.INSTANCE_ID, instance.getId());
-                mappingData.put(INSTANCE_LABEL_MAP.ACCOUNT_ID, instance.getAccountId());
-                resourceDao.createAndSchedule(InstanceLabelMap.class, objectManager.convertToPropertiesFor(InstanceLabelMap.class, mappingData));
-            }
+            labelsService.createContainerLabel(instance.getAccountId(), instance.getId(), labelKey, labelValue);
```

Comment:
Forgot to refactor this out.  No longer populating the 'name' column which still has a 255 limit.